File: JS/syntax.js
/* 
* datatypes - 
* 1.null
* 2.undefined
* 3.boolean
* 4.number
* 5.NaN eg - 5/"tanmay" 
* 5.string 
* 6.object
* 7.symbol - let temp = Symbol("Tanmay Gupta")
*/

// variuable declaration
var a = 5, 
    b = 8;
let b = 6;
const c = 7;

// Array 
let arr = ["a", "b"];
console.log(arr.length);

// loops
for(let i=0; i<1; i++) {
    console.log("for-loop");
}
let i = 0;

do {
    console.log("do-while");
}while(i<0);

while(i<1) {
    console.log("while");
    i = 1;
}

for (let a of arr) {
    console.log("of "+a);
}

for (let a in arr) {
    console.log("in "+arr[a]);
}

// conditional operators

if(true) {
    console.log(true + " if-else");
} else {
    console.log(false);
}

const condition = true;
switch (condition) {
    case true: console.log(true + " switch case");
    break;
    case false: throw false;
    break;
    default: console.log("Nothing")
}

const ans  = (5>4) ? console.log(true + " terniary operator"): console.log(false);

// function declaration
function add(a,b) {
    console.log(a+b);
}

const add_new = (a,b) => {
    console.log(a+b);
}

add(a,b);
add_new(a,b);

File: Python/syntax.py
"""
    datatypes
    1. int
    2. float
    3. complex
    4. string
    5. boolean
    7. list
    8. tupple
    9. range
    10. dict
    11. set
"""

# variable declaration 
a = 5
b = 6

# function declaration
def sum(a,b):
    print(a+b)

sum(a,b)

# conditional operators 
if True:
    print(str(True) + " if-else")
elif False:
    print(False)
else :
    print("Nothing")

# array
arr = [1,2,3]
print(len(arr))

# loops ( only 2 types in py )
i = 0
while(i<1):
    print("while-else")
    i = 1
else:
    print("else-while")

for i in range(1):
    print("for-else")
else:
    print("else-for")
